servicebroker/src/main/resources/config.properties

```diff
@@ -5,7 +5,7 @@ brokerPassword=${brokerPassword}
 # Scaling server URLs and protocol settings
 serverURIList=${serverURIList}
 apiServerURI=${apiServerURI}
-httpProtocol=${httpProtocol}
+#httpProtocol=${httpProtocol}
```

Comment:
This needs a complete check of all projects and further changes in other places.  It was commented to make sure it will not generate wrong setting when doing filtering. 

The complete remove is done through a new PR
https://github.com/cloudfoundry-incubator/app-autoscaler/pull/19